tests/testthat/_snaps/batch-chat.md

```diff
@@ -0,0 +1,37 @@
+# errors if chat/provider/prompts don't match previous run
+
+    Code
+      batch_chat(chat, prompts, path)
+    Condition
+      Error in `batch_chat()`:
+      ! provider, prompts, and user_turns don't match stored values.
+      i Do you need to pick a different `path`?
+
+# informative error for bad inputs
+
+    Code
+      batch_chat("x")
+    Condition
+      Error in `batch_chat()`:
+      ! `chat` must be a <Chat> object.
+    Code
+      batch_chat(chat_ollama)
+    Condition
+      Error in `batch_chat()`:
+      ! Batch requests are not currently supported by this provider.
+    Code
+      batch_chat(chat_openai, "a")
+    Condition
+      Error in `batch_chat()`:
+      ! `prompts` must be a list or prompt, not the string "a".
```

Comment:
Is "list or prompt" expected in the error message? Or is it supposed to be "list of prompts"?

Reply:
Hmmm, the condition is `!is.list(prompts) && !is_prompt(prompts)` so the message is technically correct albeit slightly confusing.